ddtrace/settings/profiling.py

```diff
@@ -40,7 +41,12 @@ def _derive_default_heap_sample_size(heap_config, default_heap_sample_size=1024
 
 def _is_valid_libdatadog():
     # type: () -> bool
-    return platform.machine() in ["x86_64", "aarch64"] and "glibc" in platform.libc_ver()[0]
+    return platform.system() == "Linux"
+
+
+def _is_valid_v2_stack():
+    # type: () -> bool
+    return sys.version_info >= (3, 7) and _is_valid_libdatadog()
```

Comment:
```suggestion
    return _is_valid_libdatadog()
```
We no longer support < 3.7 so this condition can never be true?

Reply:
Ah, good catch.  First of all, this check is wrong.  Second, it's no longer needed because of how we check later.  Will fix.